diff --git a/include/configs/wandboard.h b/include/configs/wandboard.h
index e21772f..bc4accc 100644
--- a/include/configs/wandboard.h
+++ b/include/configs/wandboard.h
@@ -100,7 +100,7 @@
 	"fdt_addr=0x18000000\0" \
 	"ip_dyn=yes\0" \
 	"mmcdev=" __stringify(CONFIG_SYS_MMC_ENV_DEV) "\0" \
-	"finduuid=part uuid mmc 0:1 uuid\0" \
+	"finduuid=part uuid mmc 0:2 uuid\0" \
 	"update_sd_firmware_filename=u-boot.imx\0" \
 	"update_sd_firmware=" \
 		"if test ${ip_dyn} = yes; then " \
